packages/portalnetwork/src/client/eth.ts

```diff
@@ -1,15 +1,15 @@
 import { fromHexString, toHexString } from '@chainsafe/ssz'
 import { EVM } from '@ethereumjs/evm'
 import { Address, TypeOutput, bytesToHex, hexToBytes, toType } from '@ethereumjs/util'
+import { keccak256 } from 'ethereum-cryptography/keccak.js'
```

Comment:
Can you do the imports from Util? Maybe you guys also add this listing rule to disallow ethereum-cryptography imports (bigger project: adjust *all* listing rules)?

Reply:
import `keccak256` from `@ethereumjs/util`?

Reply:
Yes

Reply:
Ah, you are right, forgot about the custom crypto thing, forget about it!